tests/test_edit.rs

```diff
@@ -450,6 +450,67 @@ fn test_sort_values() {
 "#);
 }
 
+#[test]
+fn test_multiple_positions() {
+    given(r#"
+        [package]
+        [dependencies]
+        [dependencies.opencl]
+        [dev-dependencies]"#
+    ).running(|root| {
+        for (_, table) in root.iter_mut() {
+            as_table!(table).set_position(2)
+        }
+    }).produces_in_original_order(r#"
+        [package]
+        [dependencies]
+        [dev-dependencies]
+        [dependencies.opencl]
+"#);
+}
```

Comment:
you're setting every's position to 2, could you alter the test to only do this for one table, e.g. `[dependencies]`?

Reply:
I changed it as you suggested I also set the position of the nested table.